### PR TITLE
Fix #1905, Unassigned staff error is shown in the same page

### LINKED ISSUE #1905: In productive collection sheet if staff is not assigned error message should be displayed in same page
Reported by Subramanya at https://mifosforge.jira.com/browse/MIFOSX-2237

Original Description:
In productive collection sheet if staff is not assigned error message should be displayed in same page rather than navigating to next page.

### PR DESCRIPTION
**Pull request by iamsk9**
Now In productive collection sheet if staff is not assigned error message is displayed in 
same page.

Please view the changes made and tell me whether it is working or not.

Thank You :)

### CODE REVIEW

File: app/views/collection/collectionsheet.html [hunk 1]
```diff
@@ -6,15 +6,20 @@
...
 	    translate}}</strong></h2>
-	
+
 	<h2><strong ng-show="val" class="display-save-message">{{'label.savecollectionsheetsuccessfully' |
 	    translate}}</strong></h2>
 	<fieldset ng-hide="val || noData">
 	<div>
+			<div ng-show="noStaffError">
+					<div class="alert alert-danger">
+							<strong>Staff is mandatory</strong>
```
Comment (mbj36) [inline app/views/collection/collectionsheet.html]: It should not be hard-coded

**Comment by mbj36** [comment]
@iamsk9 I see there are two accounts, you are commenting from different accounts and making changes from different account

**Comment by iamsk9** [comment]
Sorry for the hard coding. The code is nicely indented in my editor but got some extra spaces here. I fixed it. I didn't see that my git config user.email has been changed. But i changed that now. My git email id is iamsk1996@gmail.com

**Comment by nazeer1100126** [comment]
This PR have conflicts, also check whether this issue is reproducible on latest code.

**Pull request closed without merging**